### PR TITLE
Cast the SQLAlchemy db object to string

### PR DESCRIPTION
**Pull request by sunank200**
# Description
## What is the current behavior?

Make the Database SQL query as string object to make it serializable

closes: #1064 

## What is the new behavior?

- Make the Database SQL query as string object to make it serializable

## Does this introduce a breaking change?
No

### Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/src/astro/sql/operators/append.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
             target_table=self.target_table,
             source_to_target_columns_map=self.columns,
         )
-        context["ti"].xcom_push(key="append_query", value=db.sql)
+        context["ti"].xcom_push(key="append_query", value=str(db.sql))
```
Comment (kaxil) [inline python-sdk/src/astro/sql/operators/append.py:71]: Have you tested this?

Can we add a test to check the type / contents of what is pushed

**Review by dimberman — APPROVED** [review]

File: python-sdk/src/astro/sql/operators/append.py (hunk 1 above)
Comment (sunank200) [reply python-sdk/src/astro/sql/operators/append.py:71]: Yes we tested it on the DAG

Comment (pankajastro) [reply python-sdk/src/astro/sql/operators/append.py:71]: yes, 
Screenshot 2022-10-14 at 8 03 07 PM

Comment (sunank200) [reply python-sdk/src/astro/sql/operators/append.py:71]: We also checked the type and it is `str`

**Review by pankajastro — APPROVED** [review]

File: python-sdk/src/astro/sql/operators/append.py (hunk 1 above)
Comment (kaxil) [reply python-sdk/src/astro/sql/operators/append.py:71]: sounds good

Comment (kaxil) [reply python-sdk/src/astro/sql/operators/append.py:71]: We should also a add a unit-test in a follow-up PR next week

**Pull request merged**